base/bitarray.jl

```diff
@@ -235,8 +235,31 @@ function fill!(B::BitArray, x)
     return B
 end
 
-falses(args...) = fill!(BitArray(args...), false)
-trues(args...) = fill!(BitArray(args...), true)
+"""
+    falses(dims)
+
+Create a `BitArray` with all values set to `false`.
+
+    falses(A)
+
+Create a `BitArray` with all values set to `false` of the same shape as `A`.
+"""
```

Comment:
Um, yes, just tried to, but then `genstdlib.jl` does not include the docs for the second signature in its output. Any idea what I'm doing wrong? The REPL output with "?" looks the same as before.